frontend/src/components/homepage/homepageLeaderboard/homepageLeaderboard.tsx

```diff
@@ -0,0 +1,63 @@
+import { useEffect, useState } from "react";
+import {
+  getLeaderboard,
+  getPopularLocations,
+  LeaderboardEntry,
+  Location,
+} from "../../../clients/apiClients";
+import "./HomepageLeaderboard.scss";
+import { Button } from "react-bootstrap";
+import { Link } from "react-router-dom";
+
+export function HomeLeaderboard() {
+  const [topEntries, setTopEntries] = useState<Array<LeaderboardEntry>>();
+  useEffect(() => {
+    getLeaderboard().then(setTopEntries);
+  }, []);
+  if (topEntries === undefined) {
+    return <section>Loading...</section>;
+  } else
+    return (
+      <>
+        <h1 className="home-leaderboard__title">Leaderboard Top 3</h1>
+        <section className="home-leaderboard__text-container">
+          <section className="home-leaderboard__text home-leaderboard__left">
+            <ul className="home-leaderboard__list-item">
+              <li>
+                <h5 className="home-leaderboard__list-item-heading">
+                  🥇 {topEntries[0].username} with a total of{" "}
```

Comment:
Prettier has added its fun little things
```suggestion
                  🥇 {topEntries[0].username} with a total of
```

Reply:
Removed {" "}